### DISCUSSION TITLE
v1.10.1 - Maintenance Release

### DISCUSSION THREAD

**Discussion by iwalton3** [jellyfin/jellyfin-desktop#648] [General]
# This build REQUIRES Jellyfin 10.9.1 or later to work. You will get a blank home screen if you don't use a new enough server version.

Changes:

  * Upgrade jellyfin-web to 10.9.2.
  * Fix overscroll and inability to launch on MacOS (#628)
  * Update skip intro plugin.
  * Fix setBackdropTransparency.

About Windows Downloads:
 - The desktop installer requires administrator rights.
 - You can avoid warnings on files by unblocking the file in the properties dialog.
 - You can manually build a release [using these instructions](https://github.com/jellyfin/jellyfin-media-player#building-for-windows).

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin-media-player/releases/tag/v1.10.2'>v1.10.2 - Maintenance Release</a>.</em>

**Comment by tennojinesoberi** [comment]
already been updating 1.10.1 multiple times, but somehow my build will never change from 10.9.1 (at least the server is 10.9.2). since i'm not experienced  in coding, can someone help me?
![ima.ge](https://github.com/jellyfin/jellyfin-media-player/assets/162678170/af198544-575e-4088-bb9d-d2cf74c551dd)
![image](https://github.com/jellyfin/jellyfin-media-player/assets/162678170/7f10927a-fcae-4774-baf8-58fda99c2b5e)

**Comment by lonkous** [comment]
@iwalton3 idk how your cranking out new versions like there is no tomorrow keep it up  👍 👍 👍 Much appreciated

**Comment by mikewesten** [comment]
Which version do Linux Focal users download?

**Reply by majjuss** [reply]
I asked the same question in the previous release... why is there no more deb file for focal-based systems?!?! I also could not built it myself :(

**Reply by mikewesten** [reply]
The Flatpak works at least. It can be installed alongside.

**Comment by lukaslindnermusic** [comment]
Thank you 😊 The app launches now on Intel Macs. Unfortunately, the Apple Silicon build still crashes on start (but as it is not mentioned in the release notes I already assumed that it was not fixed with this update).

On the Intel build in the video player, if the video has chapters, the vertical chapter markers on the timeline disappear every second for a little amount of time. It looks like if they are refreshing with each second that passes when playing (updating with the second counter).

**Reply by lukaslindnermusic** [reply]
also, with some movies I get these icons underneath the timeline.
![Bildschirmfoto 2024-05-19 um 12 26 07](https://github.com/jellyfin/jellyfin-media-player/assets/8149031/c43b6dc5-3d19-425d-8111-7d785e883e30)

**Reply by elkenders** [reply]
Same here. Latest image on the server on docker and crashes on silicon. I'll try the intel build with Rosetta, thanks.

**Comment by michaelbi** [comment]
the sidebar in controll panel is in the wrong side when using RTL lang
![image](https://github.com/jellyfin/jellyfin-media-player/assets/1517218/b6241f45-f8ae-4295-a5d7-ee7d5f83cc1b)

**Reply by thornbill** [reply]
This is being tracked in https://github.com/jellyfin/jellyfin-web/issues/4815

**Comment by orpheasgr** [comment]
How do I update my server?
![20240523_223338](https://github.com/jellyfin/jellyfin-media-player/assets/170681332/9578182e-efe5-4127-9058-727c9d18c199)

**Comment by Gasi-Code** [comment]
I use Jellyfin directly on my Synology NAS from the Community Package Center.
But sadly the update isn't availeble yet.. is somebody know when I can update it directly in the package center? 

![2024-05-29 10_43_45-Window](https://github.com/jellyfin/jellyfin-media-player/assets/114816526/418175e3-a592-44f2-b0b1-55535404adb0)
